userapi/producers/syncapi.go

```diff
@@ -34,7 +35,7 @@ func NewSyncAPI(db storage.Database, js JetStreamPublisher, clientDataTopic stri
 }
 
 // SendAccountData sends account data to the Sync API server.
-func (p *SyncAPI) SendAccountData(userID string, roomID string, dataType string) error {
+func (p *SyncAPI) SendAccountData(userID string, roomID string, dataType string, readMarker *eventutil.ReadMarkerJSON, ignoredUsers *types.IgnoredUsers) error {
 	m := &nats.Msg{
 		Subject: p.clientDataTopic,
```

Comment:
Is this the same as `Subject: p.TopicClientData,` from clientapi?

Reply:
Yes, derived from `cfg.Matrix.JetStream.Prefixed(jetstream.OutputClientData)`.